source/features/set-default-repositories-type-to-sources.js

```diff
@@ -0,0 +1,13 @@
+import select from 'select-dom';
+
+export default function () {
+	// Get repositories link from user profile navigation
+	const link = select('.user-profile-nav:not(.is-placeholder) a[href*="tab=repositories"]');
```

Comment:
What's `is-placeholder`?

Reply:
There are 2 elements with `.user-profile-nav` class, one with `.is-placeholder` class is an empty hidden element and other contains navigation links.

When the navigation gets stuck on scroll, the placeholder element gets visible and occupies navigation's space.

Reply:
K

Reply:
But that's fine, it doesn't contain any `a` so it's filtered out anyway.

I think this code should just be:

```js
const link = select('.user-profile-nav a[href*="tab=repositories"]');

if (link) {
	link.search += '&type=source';
}
```

- It checks for the existence of the links itself
- The link we select never has a type set
- Since we're already querying `tab=repositories` we know it has a search parameter, so `+= '&...'` is enough